jsapp/scss/components/_kobo.asset-row.scss

```diff
@@ -105,20 +109,13 @@ $hover-asset-row: #F2F2F3;
 
   > .asset-row__buttons {
     position: absolute;
-    top: 8px;
-    right: 0px;
-    width: 0;
-    overflow: hidden;
-    opacity: 0;
+    top: ($s-asset-row - $s-asset-row-icon) * 0.5;
```

Comment:
I'm not sure why you changed this from a simple pixel value to a calculation. From what I can tell, the calculation results in 8px too...

Reply:
I did it because these values are connected - but I changed this top to work for rows with description (wasn't centered) and removed those unnecessary variables. I guess premature optimization :P